src/other/Table.js

```diff
@@ -1,12 +1,13 @@
 "use strict";
 (function (root, factory) {
     if (typeof define === "function" && define.amd) {
-        define(["d3", "../common/HTMLWidget", "./Paginator", "../common/Utility", "css!./Table"], factory);
+        define(["d3", "../common/HTMLWidget", "./Paginator", "../common/Utility", "../common/Widget", "css!./Table"], factory);
```

Comment:
OK, very good.  I was careful (to at least try) not to create any circular references.  If this is the best way, I'll leave it ass is except for the following comments...